corehq/apps/dashboard/models.py

```diff
@@ -317,13 +317,7 @@ def total(self):
     @property
     @memoized
     def applications(self):
-        key = [self.request.domain]
-        return Application.get_db().view(
-            'app_manager/applications_brief',
-            reduce=False,
-            startkey=key,
-            endkey=key+[{}],
-        ).all()
+        return get_brief_apps_in_domain(self.request.domain)
```

Comment:
@benrudolph oops this change wasn't quite equivalent and broke something. Fixed here: https://github.com/dimagi/commcare-hq/pull/9953